index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <nav id="navbar">
+      <header>Js Documentation</header>
+      <a class="nav-link" href="#Introduction" >Introduction</a>
+      <a class="nav-link" href="#What_you_should_already_know" >What you should already know</a>
+      <a class="nav-link" href="#JavaScript_and_Java" >JavaScript and Java</a>
+      <a class="nav-link" href="#Hello_world" >Hello world</a>
+      <a class="nav-link" href="#Variables" >Variables</a>
+
+    </nav>
+  <main id="main-doc">
+    <section class="main-section" id="Introduction">
+      <header>Introduction</header>
+      <article>
+        <p>This is first paragraph</p>
+        <p>This is second paragraph</p>
+        <li>Client-side JavaScript extends the core language by supplying objects to control a browser and its Document Object Model (DOM).</li>
+        <code>const MY_OBJECT = {"key": "value"}; MY_OBJECT.key =
+          "otherValue";</code>
+      </article>
+    </section>
+    <section class="main-section" id="What_you_should_already_know">
+      <header>What you should already know</header>
```

Comment:
You can use hx that is h1-h6 here

Reply:
I suggest instead of using a header tag kindly use a hx tag that is h1-h6 depending on the size you want